olive/systems/azureml/aml_system.py

```diff
@@ -125,14 +126,70 @@ def __init__(
         self.env_vars = self._get_hf_token_env(self.azureml_client_config.keyvault_name) if self.hf_token else None
         self.temp_dirs = []
 
+    def submit_workflow(self, workflow_id: str):
```

Comment:
how does someone retrieve the output of the workflow? Do they have to go to the datastore and download the files manually?

Reply:
Yes, I got some errors when downloading from datastore by azure file system api. So I didn't add retrieve feature for now.